scripts.js

```diff
@@ -6,8 +6,17 @@
 // 2. Look for the @TODOs, and figure out how to fix them.
     // next to each @TODO you will find tasks that need to be finished
 
+   
+
+
 // The variable will change from X to O based on what player turn it is. We need to hold this so we can place an X or O on the board when they're clicked.
 let currentMarker = 'X'
+let Board = [
+ 
```

Comment:
you have capital Board here but in the checkForWin function you are checking for lowercase board. either change those to Capital Board or lowercase this one to match